pkg/virtual/workspaces/registry/rest.go

```diff
@@ -215,17 +207,22 @@ func withoutGroupsWhenPersonal(user user.Info, scope string) user.Info {
 func (s *REST) extractOrg(user user.Info, ctx context.Context) (orgClusterName string, org *Org, err error) {
 	orgClusterName = ctx.Value(WorkspacesOrgKey).(string)
 
-	// Any user must has at least access to the content of the root org, in order to
-	// be able to fetch the list of the orgs he is member of
+	// Any user must have at least access to the content of the root org, in order to
+	// be able to fetch the list of the orgs they're a member of
 	if orgClusterName != helper.RootCluster {
-		reviewer := s.rootReviewerProvider.Create("access", "clusterworkspaces", "content")
 		_, orgName, err := helper.ParseLogicalClusterName(orgClusterName)
 		if err != nil {
 			return "", nil, kerrors.NewBadRequest("unable to determine organization")
 		}
-		if allowed, err := s.isUserAllowed(user, orgName, reviewer); err != nil {
-			return "", nil, kerrors.NewForbidden(tenancyv1beta1.Resource("workspaces"), "", err)
-		} else if !allowed {
+
+		review := s.rootReviewer.Review(
+			workspaceauth.NewAttributesBuilder().
+				Verb("access").
+				Resource(tenancyv1alpha1.SchemeGroupVersion.WithResource("clusterworkspaces"), "content").
+				Name(orgName).
+				AttributesRecord,
+		)
+		if !review.Includes(user) {
```

Comment:
we cannot do this: I was not aware that the reviewer gives us a list of users.

1. this does not scale
2. users will not be the subjects of ClusterRoleBindings in the root org, but OIDC groups will be.

Reply:
blocker 🔴

Reply:
`review.Includes(user)` is misleading. It includes group checks. Maybe better: `review.Allows(user)`. "include" is about identity.